src/maintenance/performance.rst

```diff
@@ -212,6 +212,21 @@ This TCP buffering behaviour can be disabled via
 .. seealso::
     Bulk :ref:`load <api/db/all_docs>` and :ref:`store <api/db/bulk_docs>` API.
 
+Connection limit
+----------------
+
+`MochiWeb`_ is handling CouchDB requests.
+By default maximum number of connections is 2048. To change this limit, use
```

Comment:
This sentence is ungrammatical. I suggest either "The default maximum number of connections is..." or "By default, the maximum number of connections is..."